README.md

```diff
@@ -1,11 +1,52 @@
-## My Project
+[![License](https://img.shields.io/hexpm/l/plug.svg)](https://github.com/partiql/homebrew-partiql/blob/main/LICENSE)
+[![Homebrew](https://img.shields.io/badge/Homebrew-Install-brightgreen.svg)](https://github.com/partiql/homebrew-partiql)
+## Homebrew PartiQL
 
-TODO: Fill this README out!
+Welcome to the official Homebrew Tap for PartiQL applications.
 
-Be sure to:
+This repository is managed by the [PartiQL](https://github.com/partiql) team and contains Homebrew formulas for easy installation and management of PartiQL owned applications on macOS and Linux systems.
 
-* Change the title in this README
-* Edit your repository description on GitHub
+### Installation
+To tap this repository and make the formulas available on your system, run: 
+
+```bash
+brew tap partiql/partiql 
+```
+
+### Usage
+Once the tap is added, you can install PartiQL applications using the `brew install` command. For example:
```

Comment:
Since we don't have plans to add more formulas (yet), we can simplify the instructions a bit.

````suggestion
We currently publish one formula, `partiql-cli`. To install, run:
```
brew install partiql-cli
```
````